apps/converter/src/OnclickFunc.js

```diff
@@ -0,0 +1,28 @@
+import formule from '../data/formula';
+
+function OnclickFunc() {
+  let index;
+  const topSelectionVal = document.querySelector('#mySelect').value;
```

Comment:
You should use DOM in react projects in exceptional cases and handle data by state and props. You Use DOM here too much and it doesn't make sense. Perhaps you should reconsider implementing this function by lifting the state up from the Selection component to the App component and then passing them as props where you need them.